modules/modificadores/modificadores.routes.ts

```diff
@@ -0,0 +1,35 @@
+import * as express from 'express';
+import { asyncHandler } from '@andes/api-tool';
+import { Auth } from '../../auth/auth.class';
+import { Modificadores } from './modificadores.schema';
+import { engine } from './generadorModificadores';
+
+const router = express.Router();
+
+router.post('/modificadores', Auth.authenticate(), asyncHandler(async (req: any, res) => {
+    let response: any = {};
+    if (req.body.prestacion && req.body.paciente) {
+        try {
+            const paciente = req.body.paciente;
+            const prestacion = req.body.prestacion;
+            const modificador: any = await Modificadores.find({ conceptId: prestacion.conceptId });
+            if (modificador.length) {
+                const condiciones = modificador[0].condicionesPaciente;
```

Comment:
```suggestion
                const condiciones = modificador[0].condiciones;
```
Sería más genérico ¿siempre son condiciones de pacientes? ¿pueden ser condiciones de otra indole??

Reply:
Si, en principio las condiciones las debe cumplir el paciente que solicita el turno. Quizás más adelante cambie.